include/wx/pdfdc29.h

```diff
@@ -231,6 +231,12 @@ class WXDLLIMPEXP_PDFDOC wxPdfDCImpl: public wxDCImpl
   wxPrintData    m_printData;
   wxPdfMapModeStyle m_mappingModeStyle;
 
+  bool          m_cachedColourSet{false};
```

Comment:
wxPdfDoc still supports C++98 compilers, so we can't use C++11 member initialization in it, please change this.

On second thought, do we actually need this one at all? Default initialized `wxPdfColour` can already be distinguished from any valid colour by checking for `GetColourType() == wxPDF_COLOURTYPE_UNKNOWN`, so I don't think we do.